docker.txt

```diff
@@ -0,0 +1,5 @@
+gunicorn==19.7.1
```

Comment:
Why to create new file, you could reuse `requirements.txt`

Reply:
`Dockerfile` was giving error while executing when I used `requirements.txt`, that's why I have added `docker.txt`.

Reply:
Now I have made the changes @vaibhavsingh97 , I have to revert all the changes for reproducing it.